extensions/wikia/AdEngine/js/provider/directGptMobile.js

```diff
@@ -15,7 +15,7 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 			INVISIBLE_HIGH_IMPACT:      {size: '1x1'},
 			INVISIBLE_HIGH_IMPACT_2:    {loc: 'hivi'},
 			MOBILE_TOP_LEADERBOARD:     {size: '300x50,320x50,320x100,320x480,2x2', loc: 'top'},
-			MOBILE_BOTTOM_LEADERBOARD:  {size: '320x480,2x2', loc: 'footer'},
+			MOBILE_BOTTOM_LEADERBOARD:  {size: '320x50,300x250,300x50,2x2', loc: 'footer'},
```

Comment:
Should we add BOTTOM_LEADERBOARD at this point?

Reply:
If yes... we need to handle it separately in slotContext based on skin

Reply:
Added